### ISSUE TITLE
Background colour

### ISSUE THREAD

**Issue by djensenius** [christopher-francisco/tmux-status.nvim#4]
Hello-

When using lualine the background colour doesn't match. 

I made this change to make it work.

```
local M = {}
local utils = require("lualine.utils.utils")

---@param colors TmuxStatusComponentColors
function M.create_hightlights(colors)
  local lualine_bg = utils.extract_highlight_colors("statusline", "bg")
  for name, value in pairs(colors) do
    vim.cmd.highlight('tmux_status_' .. name .. ' guifg=' .. value .. ' guibg=' .. lualine_bg)
  end
end

---@param name string
---@return string
function M.get_highlight(name)
  return "%#" .. name .. "#"
end

return M
```

**Comment by christopher-francisco** [comment]
Nice one. Looks like you'd like the ability to set  the BG color, so that it's retained. Perhaps this is better passed down in the config object. Do you mind sharing a couple of screenshots, 1 without the change above, and one with it?

**Comment by djensenius** [comment]
Sure thing!

Without:
![Image](https://github.com/user-attachments/assets/809fe2fe-6144-4654-a2a8-cc6015f76523)

With:
![Image](https://github.com/user-attachments/assets/51d32240-d10d-489d-a103-3db608dc0ca3)

Using `theme = "catppuccin"`  in lualine

**Comment by christopher-francisco** [comment]
Makes sense. Perhaps the way to go is expanding the properties in `colors` to take an object defining both fg and bg, as opposed to a string. Do you think that could work?

**Comment by djensenius** [comment]
That could work. :)

**Comment by RileyGabrielson** [comment]
Bump on this, seems like the PR mentioned would solve this, any reason not to merge? @christopher-francisco

**Comment by christopher-francisco** [comment]
Approved! Apologies for the delay, I didn't notice the PR was raised. Thanks for letting me know @RileyGabrielson.

**Comment by RileyGabrielson** [comment]
No worries! Love this plugin, thanks for taking a look
